inspectit-oce-core/src/main/resources/config/default.yml

```diff
@@ -107,6 +107,21 @@ inspectit:
       # defines how often after the last invocation of a ClassFileTransformer the agent scans for new classes
       num-class-discovery-trials: 2
 
+    #definition of commonly used data providers
+    data-providers:
+
+      #computes a nanosecond-timestamp as a long for the current point in time
+      timestampNanos:
+        value: "new Long(System.nanoTime())"
```

Comment:
So we always need to work with objects right? that's because we are returning the `Object` as the result of the data providers.. I am only thinking that we are making a lot of garbage this way, but anyway we would be wrapping this into some kind of map or list, thus we would anyway need an object.. Right?

Reply:
Yes we store everything in a map, that's why we need an object anyway. To my knowledge constructing small objects with a short livetime isn't that much of a problem in modern JVMs anymore..